Qq/MetaM.lean

```diff
@@ -85,3 +85,26 @@ def assertDefEqQ {α : Q(Sort $u)} (a b : Q($α)) : MetaM (PLift (QuotedDefEq a
   match ← isDefEqQ a b with
   | .defEq witness => return ⟨witness⟩
   | .notDefEq => throwError "{a} is not definitionally equal to{indentExpr b}"
+
+/-- The result of `Qq.isLevelDefEqQ`; `MaybeLevelDefEq u v` is an optional version of `$u =QL $v`. -/
+inductive MaybeLevelDefEq (u v : Level) where
+  | defEq : QuotedLevelDefEq u v → MaybeLevelDefEq u v
+  | notDefEq : MaybeLevelDefEq u v
```

Comment:
I guess you're just following `MaybeDefEq` but why not use `Option (QuotedLevelDefEq u v)`?

Reply:
Maybe @gebner has an opinion?